src/components/navbar.js

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { Link } from "gatsby"
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  Nav,
+  NavItem,
+  NavLink,
+} from "reactstrap"
+
+import Logo from "../images/svg/logo_sinf_simp.inline.svg"
+import NavbarStyles from "../styles/navbar.module.css"
+
+// Markup for navbar links redirecting to an internal page
+const InternalNavLink = ({ to, text }) => (
+  <Link
+    className={`nav-link ${NavbarStyles.navLink}`}
+    activeClassName={NavbarStyles.navLinkActive}
+    to={to}
+  >
+    {text}
+  </Link>
+)
+
+export default class Example extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.toggle = this.toggle.bind(this)
+    this.state = {
+      isOpen: false,
+    }
+  }
+  toggle() {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    })
+  }
+  render() {
+    return (
+      <div>
+        <Navbar dark fixed="top" expand="md" className={NavbarStyles.navbar}>
+          <Collapse
+            isOpen={this.state.isOpen}
+            navbar
+            className=" w-100 order-1 order-md-0 dual-collapse2"
+          >
+            <Nav navbar className="mr-auto ml-0">
```

Comment:
It is possible to refactor the contents of this nav to be a `.map()` over an array of objects 😉 
Makes for cleaner and more modular code!

Would be something like
```jsx
{[
  {url: "/stuff", text: "stuff and things"},
  {url: "/stuff2", text: "stuff and things 2"},
].map(elem => (
  <InternalNavLink to={elem.url} text={elem.text}/>
))}
```

This probably even disregards the `InternalNavLink` component altogether, as a boolean attribute for a link being internal or not can also be included in the object array and be checked there, making for cleaner code overall imo.